environment.yml

```diff
@@ -10,6 +10,7 @@ dependencies:
   - matplotlib<3
   - python-stratify
   # Multi language support:
+  - gcc
```

Comment:
Please add this also to meta.yml (the conda package recipe)

Reply:
I wonder why you need gcc

Reply:
Lot of R packages compile Fortran and C/C++ code. We need gcc to install those

Reply:
This will close https://github.com/ESMValGroup/ESMValTool/issues/628

Reply:
I hope so

Reply:
> Lot of R packages compile Fortran and C/C++ code. We need gcc to install those

I think, you want to add r-rcpp package from conda-forge instead of gcc which may cause some other problems.

Reply:
> This will close #628, or not?

Yes, we can close #628.